### PR TITLE
apache and apr-util cross

### PR DESCRIPTION
**Pull request by Gottox**
fixes apr-util and apache cross compilation.

### CODE REVIEW

File: srcpkgs/apache/template [hunk 1]
```diff
@@ -45,6 +45,27 @@ pre_configure() {
 		-i docs/conf/httpd.conf.in
 
 	cat ${FILESDIR}/xbps.layout >> config.layout
+
+	if [ "$CROSS_BUILD" ]; then
+		configure_args+=" --with-apr=${XBPS_WRAPPERDIR}/apr-1-config
+			 --with-pcre=${XBPS_WRAPPER_DIR}/pcre-config"
```
Comment (xtraeme) [inline srcpkgs/apache/template]: this should be unnecessary, XBPS_WRAPPERDIR appears in PATH before /usr/bin.

File: srcpkgs/apr-util/template [hunk 2]
```diff
@@ -19,7 +18,17 @@ license="Apache-2.0"
...
 checksum=a6cf327189ca0df2fb9d5633d7326c460fe2b61684745fd7963e79a6dd0dc82e
 
-nocross="http://build.voidlinux.eu/builders/armv7l_builder/builds/7623/steps/shell_4/logs/stdio"
+pre_configure() {
+	if [ "$CROSS_BUILD" ]; then
+		configure_args+=" --with-apr=${XBPS_WRAPPERDIR}/apr-1-config"
+	else
+		configure_args+=" --with-apr=/usr/bin/apr-1-config"
+	fi
+}
```
Comment (xtraeme) [inline srcpkgs/apr-util/template]: This is again unnecessary, apr-1-config in XBPS_WRAPPERDIR should be found via PATH automagically.

Comment (Gottox) [inline srcpkgs/apr-util/template]: without it:
```
configure: error: APR could not be located. Please use the --with-apr option.
```
The build system is broken.

File: srcpkgs/apache/template (hunk 1 above)
Comment (Gottox) [inline srcpkgs/apache/template]: doesn't work. the buildsystem is broken.

**Comment by asergi** [comment]
@pullmoll added cross support to `apr-util` and `apache` with 3dcf63b209d6c33f40eec481c41e1220d773d574 and aad6395993278b7227418ac1f76146c04de127af.

**Pull request closed without merging**